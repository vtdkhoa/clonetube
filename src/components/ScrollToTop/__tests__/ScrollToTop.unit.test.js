import React from 'react'
import ScrollToTop from '../ScrollToTop'
import { shallow } from 'enzyme'

describe('ScrollToTop', () => {
  test('renders without children props', () => {
    const wrapper = shallow(<ScrollToTop/>)
    expect(wrapper).toMatchSnapshot()
  })

  test('renders with children props', () => {
    const wrapper = shallow(
      <ScrollToTop>
        <div>Children Props</div>
      </ScrollToTop>
    )
    expect(wrapper).toMatchSnapshot()
  })
})