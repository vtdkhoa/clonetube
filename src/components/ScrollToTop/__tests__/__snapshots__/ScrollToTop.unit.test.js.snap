// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop renders with children props 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`ScrollToTop renders without children props 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
